source/Engine.cpp

```diff
@@ -790,6 +790,20 @@ void Engine::Step(bool isActive)
 			info.SetCondition("flagship solar display");
 			info.SetString("flagship solar", to_string(flagshipSolar));
 		}
+		// Display combined Solar Power for system
+		int displaySystemPower = (flagship->DisplaySystemSolar() * 100); // Multiplied to better display decimal inputs
+		if(displaySystemPower >= 0.01 && flagship->Attributes().Get("solar scan"))
```

Comment:
This attribute name works, but I'm just thinking that it might be better to separate it from the targeting-ships scans so people don't think that they're used to actually scan stars.

Maybe "solar sensor" ?